### ISSUE TITLE
After upgrade to 5.0.3, throws error "No form with ID <formId> is currently rendered..."

### ISSUE THREAD

**Issue by delgermurun** [aldeed/meteor-autoform#801]
On 5.0.2 it was fine. After update to 5.0.3. throws error with message:

`Uncaught Error: No form with ID <formId> is currently rendered. If this call originated from within a template event or helper, you should call without passing a formId to avoid seeing this error`

How can I fix it?

**Comment by aldeed** [comment]
Can you post the whole stack trace? Are you calling something and passing in formId as it says?

**Comment by delgermurun** [comment]
Sure here is it.
```
AutoForm.viewForForm                                        autoform-api.js:900
AutoForm.templateInstanceForForm                            autoform-api.js:883
(anonymous function)                                        afFieldInput.js:72
```

I am not calling something. I think it initiated from https://github.com/aldeed/meteor-autoform/blob/master/components/afFieldInput/afFieldInput.js#L72

**Comment by aldeed** [comment]
OK, this will be fixed when I fix #789

**Comment by delgermurun** [comment]
Okay. Thanks
